### ISSUE TITLE
async mode not working over uca-net

### ISSUE THREAD

**Issue by MarcusZuber** [ufo-kit/uca-net#2]
In the async the acquisition takes for ever without returning frames.

On windows:
`ucad mock`
On linux client:
`uca-benchmark -p host=server:8989 -r 1 -n 20 net`

The resulting output is:
```
sync auto 18.7 Hz 4.5 MB/s 20/20 acquired (0.00% dropped)
async auto 1/1
```

Here it is stuck (for at least a few minutes).

**Comment by matze** [comment]
This is true. The question is: is real asynchronous acquisition necessary for you? Synchronous-only  acquisition simplifies the network code quite a bit because the server is acting like a normal server (i.e. request-reply style). Asynchronous acquisition mode would mean that the server is notifying the client that new data is available.

However, I could of course implement fake asynchronous acquisition on top of the synchronous acquisition, which in fact the mock camera also does.

**Comment by MarcusZuber** [comment]
For me it is not very important.
I just thought that you want to have the net-camera completely identical to a local one.

**Comment by matze** [comment]
Alright, then I will just do it as in the mock camera and build async mode on top of synchronous.
